elegant/segment_images.py

```diff
@@ -140,6 +147,47 @@ def annotate_poses_from_masks(positions, mask_root, annotations, overwrite_exist
                 current_annotation[annotation] = pose
             current_annotation[original_annotation] = pose
 
+def gmm_lawn_maker(image, optocoupler, return_model=False):
```

Comment:
*super minor* style point: names should simply describe what a function does, not how it does it. So maybe just call the function `find_lawn` or something (the GMM stuff is an implementation detail, not a key part of the external interface).